uber/site_sections/preregistration.py

```diff
@@ -497,7 +499,7 @@ def invalidate(self, session, id):
     def badge_updated(self, session, id, message=''):
         return {'id': id, 'message': message}
 
-    @attendee_id_required
+    @id_required(Attendee)
```

Comment:
hey only quick note, these used to say ```sa.Attendee``` instead of just ```Attendee```. It seems like either way might work, just wanted to make sure